src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -27,22 +29,30 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html");
-
     UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserLoggedIn()) {
-      String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/login";
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+    boolean loginStatus = userService.isUserLoggedIn();
+    String urlToRedirect = "/index.html";
 
-      response.getWriter().println("<p>Hello " + userEmail + "!</p>");
-      response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+    ArrayList<String> userDetails = new ArrayList();
+    if (loginStatus) {
+      userDetails.add("true");
+      String userEmail = userService.getCurrentUser().getEmail();
+      String logoutURL = userService.createLogoutURL(urlToRedirect);
+      userDetails.add(
+          "<p>Hello, "
+              + userEmail
```

Comment:
If you just added the email and logout URL without the extra html, it could be easier to do the HTML stuff in the JavaScript and would make the parsing a little easier in the JavaScript.

Reply:
Hmm, good point. I wanted to push this as a sign that I have something working, but as I add more stuff to this page, I think I'll definitely make userDetails have more elements and each of them be a separate thing.

Reply:
^ just kidding, decided to implement this because it'll make it easier in future